docs/UserGuide.md

```diff
@@ -166,7 +215,11 @@ Format: `addDes INDEX d/DESCRIPTION`
 Examples:
 `addDes 3 d/movies` Add the description field `movies` to the 3rd expense.
 
-#### Delete description of an expense: `deleteDes`
+Navigate back to the feature list: [3. Features](#3-features)
```

Comment:
What is this line? I saw it duplicated for a few times

Reply:
it go back to the Main feature